fineract-e2e-tests-core/src/test/resources/fineract-test-application.properties

```diff
@@ -23,7 +23,7 @@ fineract-test.api.password=${TEST_PASSWORD:password}
 fineract-test.api.strong-password=${TEST_STRONG_PASSWORD:A1b2c3d4e5f$}
 fineract-test.api.tenant-id=${TEST_TENANT_ID:default}
 
-fineract-test.initialization.enabled=${INITIALIZATION_ENABLED:false}
+fineract-test.initialization.enabled=${INITIALIZATION_ENABLED:true}
```

Comment:
Can you pls explain the purpose of this change?

Reply:
Integration tests can not run without initialization, that is why I've changed the default value. But I can write it back for now.